### ISSUE TITLE
Support for websocket

### ISSUE THREAD

**Issue by CharlesLim17** [usebruno/bruno#1345]
Nice open source project!

Looking forward for function that will support testing of websocket!

**Comment by pekiZG** [comment]
Linked issue:
https://github.com/usebruno/bruno/issues/316

**Comment by natalie2597-web** [comment]
Is websocket support will be for Golden edition only? Its mentioned here https://www.usebruno.com/pricing

**Comment by Pithikos** [comment]
> Is websocket support will be for Golden edition only? Its mentioned here https://www.usebruno.com/pricing If so, it's going to be a hard sell compared to, say, Insomnium

Nonsense. Doubt a developer can't afford a one-time payment of 19$ to ease their life.

But yeah, would be great if we had a timeline for websocket support

**Comment by pedrolaranjeiro** [comment]
Websocket is the only thing keeping me using Postman. I will happily pay for Golden when the websocket feature is available.

**Comment by invaderb** [comment]
Same here I would love to use it on a opensource project I'm working on

**Comment by ProjectInfinity** [comment]
Bought Bruno Golden Edition for the planned websocket support, would be great to see an update or at least a roadmap for it to be implemented.

**Comment by dylandamsma** [comment]
Yes +10. Please add web socket support. It's keeping me stuck to Insomnia/Postman right now.

**Comment by LohanGuedes** [comment]
Waiting to buy when websocket support is available for Golden Edition.

**Comment by amunrarara** [comment]
I went ahead and bought Golden because Bruno rocks. But the lack of WebSocket support _is_ a bummer. +1

**Comment by aristotekean** [comment]
I really like Bruno but without websocket support I can't buy the golden version of it.

**Comment by bytefluxio** [comment]
Did they change the pricing options? Because websocket support is a must have for me and a one time payment wouldn't be an issue at all, but I can only find subscription models, which I try to keep to a minimum for private use.

**Comment by kreibaum** [comment]
@bytefluxio Looks like websocket support moves to the free tier (when done) and the one time $20 payment option gets removed. See https://github.com/usebruno/bruno/discussions/269#discussioncomment-11352541

**Comment by nosovk** [comment]
It would be nice to have parity vs postman in that feature

**Comment by akbarisroilov** [comment]
cant wait to see the best api client to have this feature

**Comment by helloanoop** [comment]
🚀 Big News: WebSockets support has landed in Bruno v2.13.0! 🥳✨

<img width="1299" height="668" alt="Image" src="https://github.com/user-attachments/assets/b7601d91-2512-4182-91c7-afa2a56d2ce9" />

**Comment by akbarisroilov** [comment]
Thank you for your service 🫡
